packages/components/src/internal/query/api.ts

```diff
@@ -1010,24 +1005,14 @@ export function getContainerFilterForLookups(): Query.ContainerFilter {
     return Query.ContainerFilter.currentPlusProjectAndShared;
 }
 
-export interface SelectDistinctResponse {
-    queryName: string;
-    schemaName: string;
-    values: any[];
-}
-
-export function selectDistinctRows(
-    selectDistinctOptions: Query.SelectDistinctOptions
-): Promise<SelectDistinctResponse> {
+export function selectDistinctRows(options: Query.SelectDistinctOptions): Promise<Query.SelectDistinctResponse> {
     return new Promise((resolve, reject) => {
         Query.selectDistinctRows({
-            ...selectDistinctOptions,
+            method: 'POST',
+            ...options,
+            containerFilter: options.containerFilter ?? getContainerFilter(options.containerPath),
             success: response => {
-                resolve({
-                    values: response['values'],
-                    schemaName: response['schemaName'],
-                    queryName: response['queryName'],
-                });
+                resolve(response);
```

Comment:
huh, I wonder why it was previously just pulling out those three values? are there other props/values on the `response` object here? if so, I assume it is fine to include them.